pkgs/development/tools/rbspy/default.nix

```
@@ -1,16 +1,16 @@
 {rustPlatform, fetchFromGitHub, lib}:
```

> Arguments should be listed in the order they are used, with the exception of lib, which always goes first.

See: https://nixos.org/manual/nixpkgs/unstable/#sec-syntax
```suggestion
{ lib, rustPlatform, fetchFromGitHub }:
```